contracts/contracts/IERCX.sol

```diff
@@ -33,6 +33,8 @@ interface IERCX {
     struct Build {
         string commitHash;
         string gitRepository;
+        string ipfsHash;
```

Comment:
we can check how to store ipfs hash in a fixed data structure later https://ethereum.stackexchange.com/a/17112